src/ignitions.R

```diff
@@ -90,6 +91,25 @@ if(nrow(ResampleOption) == 0) {
   saveDatasheet(myScenario, ResampleOption, "burnP3Plus_FireResampleOption")
 }
 
+if(any(is.na(ProbabilisticIgnitionLocation$IgnitionGridFileName))) {
+  stop("Not all Probabilistic Ignition Grids specified in the Ignition Location datasheet.")
+}
+
+# Check for ignition count distribution
```

Comment:
I initially added this validation to that part of the code, but for some reason the error message in the `stop()` wasn't being printed to the run log, and instead a weird nondescript error was being output (maybe because the error was happening from within a pipe function?). I ended up adding it here instead so it would catch the error in the inputs right away before running the rest of the script.

Reply:
Oh weird, maybe something in the code in between was trying to use the problematic inputs before it got to the check? We could also move the distribution parsing up here if we need to. I think it's fine to leave this as is then, but the scripts are quite long and it's maybe harder than it needs to be to find where things are happening. Ideally we refactor a lot of this stuff into more modular pieces, but that might be a big task and lower priority.

Reply:
I agree, maybe when there is more time and funding we can do a bit of a refactor and fix this up.